<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsPage</class>
 <widget class="QDialog" name="SettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>350</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelUsername">
     <property name="text">
      <string>Username: user123</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="pageMenu">
      <layout class="QVBoxLayout" name="menuLayout">
       <item>
        <widget class="QPushButton" name="btnChangePassword">
         <property name="text">
          <string>Change Password</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnViewHistory">
         <property name="text">
          <string>View Purchase History</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageChangePassword">
      <layout class="QVBoxLayout" name="passwordLayout">
       <item>
        <widget class="QLabel" name="labelOldPassword">
         <property name="text">
          <string>Old Password:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineOldPassword">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelNewPassword">
         <property name="text">
          <string>New Password:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineNewPassword">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelConfirmPassword">
         <property name="text">
          <string>Confirm Password:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineConfirmPassword">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkShowPassword">
         <property name="text">
          <string>Show Passwords</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="passwordButtonsLayout">
         <item>
          <widget class="QPushButton" name="btnBackPassword">
           <property name="text">
            <string>Back</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSubmitPassword">
           <property name="text">
            <string>Submit</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pagePurchaseHistory">
      <layout class="QVBoxLayout" name="historyLayout">
       <item>
        <widget class="QTableWidget" name="tablePurchaseHistory"/>
       </item>
       <item>
        <widget class="QPushButton" name="btnBackHistory">
         <property name="text">
          <string>Back</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
